hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -571,15 +571,20 @@ public final void readData(ObjectDataInput in) throws IOException {
 
     protected abstract void readInternal(ObjectDataInput in) throws IOException;
 
+    protected void toString(StringBuilder sb) {
+    }
+
```

Comment:
There's still a slight problem here. Let's look at the hierarchy again:
```
CacheGetAllOperation extends PartitionWideCacheOperation
PartitionWideCacheOperation extends AbstractNamedOperation
```

I don't see PartitionWideCacheOperation in this PR, which means it probably doesn't implement the `toString(StringBuilder)` method. It means that it uses the default one from `Operation` - which is empty. 
Thus, it will break the chain of invocations of the toString() hierarchy and `AbstractNamedOperation.toString(StringBuilder)` will never be called.

Possible solution: making this method abstract? Then it's sure that very op implements this method.
But we still need a convention to call super.toString(sb) inside...